inst/templates/methods-nab/nab-statistical-methods.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+title: "Statistical Methods"
+output: github_document
+---
+
+<!-- This file was adapted from VISCtemplates {{pkg_ver}}. -->
+
+## Statistical Endpoints
+
+<!-- Animal studies usually use SLA-MLV-subtracted values. Check the lab study plan
+for more info. -->
+
+Response to a pseudovirus was considered to be positive if the net ID~50~ titer 
+was greater than or equal to 20.
+The net ID~50~ titer (the SLA-MLV-subtracted neutralization titer) is the 
+pseudovirus ID~50~ titer value minus the SLA-MLV ID~50~ titer value, calculated 
+for each animal and timepoint.
+SLA-MLV ID~50~ titer values less than 20 were truncated at 10 before subtraction.  
+Response rates were calculated for each group and time point with corresponding 
+95% two-sided Wilson score confidence intervals.
+
+The response magnitude was defined as the log~10~ net ID~50~ titer.
+
+To measure durability, the fold-change in response magnitude (log~10~ net ID~50~ 
+titer) between week [peak] (peak time point) and week [post-peak] was calculated. 
+The fold-change was only calculated for pseudoviruses with samples at both time 
+points and for animals with positive responses at 
+week [peak], <!--usually first sample taken after last vaccination --> 
+defined as:
+
+$$\mathrm{Fold\mbox{-}change} = \frac{X_{\text{week [peak]}}}{X_{\text{week [post-peak]}}}$$
+
+where X represents the response magnitude.
+
+## Graphical Analysis
+
+Response rates were plotted, with accompanying Wilson score confidence 
```

Comment:
The template contains text for a comprehensive unblinded report. Rather than add brackets ("[Unblinded report]") next to optional things, I think we should leave it to the SRA to decide on which parts to edit and use, and which parts to remove, based on the SAP and report requirements.